### ISSUE TITLE
Save command fail.

### ISSUE THREAD

**Issue by ma8ma** [eagletmt/comic_walker#6]
After [site update](http://comic-walker.com/news/detail/383/) at 2015-12-17, ``comic-walker save cid`` fail.

build log and error message:
```bash
# Ubuntu 14.04
$ ruby --version
ruby 1.9.3p484 (2013-11-22 revision 43786) [i686-linux]
$ git clone https://github.com/eagletmt/comic_walker
$ cd comic_walker
$ echo "layout ruby" >.envrc
$ direnv allow
$ bundle
$ bundle exec comic-walker save KDCW_EB02000004010001_68
/path/to/comic_walker/lib/comic_walker/client.rb:27:in `get_license': 400: Unexpected request parameters (ComicWalker::Client::Error)
	from /path/to/comic_walker/lib/comic_walker/content_downloader.rb:9:in `save'
	from /path/to/comic_walker/lib/comic_walker/cli.rb:23:in `block in save'
	from /path/to/comic_walker/lib/comic_walker/cli.rb:22:in `each'
	from /path/to/comic_walker/lib/comic_walker/cli.rb:22:in `save'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /path/to/comic_walker/bin/comic-walker:4:in `<top (required)>'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/bin/comic-walker:23:in `load'
	from /path/to/comic_walker/.direnv/ruby-ruby-1.9.3/bin/comic-walker:23:in `<main>'
```

**Comment by ma8ma** [comment]
In [ComicWalker::Client::get_u1(cid)](../blob/e1d2801de7afd9960036ae22578a34d1053b74b6/lib/comic_walker/client.rb#L34),
``http://comic-walker.com/viewer/?cid=#{cid}`` returns 404 Not Found.
Request URL should modify to ``http://comic-walker.com/viewer/?tw=2&dlcl=ja&cid=#{cid}``.

**Comment by eagletmt** [comment]
Thank you!
I've just released v0.4.0 including the fix :gem:
